### ISSUE TITLE
OpenJS Foundation Cross Project Council Meeting 2019-07-30

### ISSUE THREAD

**Issue by mhdawson** [openjs-foundation/cross-project-council#272]
## Time

**UTC Tue 30-Jul-2019 16:00 (04:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Tue 30-Jul-2019 09:00 (09:00 AM) |
| US / Mountain | Tue 30-Jul-2019 10:00 (10:00 AM) |
| US / Central  | Tue 30-Jul-2019 11:00 (11:00 AM) |
| US / Eastern  | Tue 30-Jul-2019 12:00 (12:00 PM) |
| London        | Tue 30-Jul-2019 17:00 (05:00 PM) |
| Amsterdam     | Tue 30-Jul-2019 18:00 (06:00 PM) |
| Moscow        | Tue 30-Jul-2019 19:00 (07:00 PM) |
| Chennai       | Tue 30-Jul-2019 21:30 (09:30 PM) |
| Hangzhou      | Wed 31-Jul-2019 00:00 (12:00 AM) |
| Tokyo         | Wed 31-Jul-2019 01:00 (01:00 AM) |
| Sydney        | Wed 31-Jul-2019 02:00 (02:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Cross%20Project%20Council+Meeting+2019-07-30&iso=20190730T16
* or http://www.wolframalpha.com/input/?i=04PM+UTC%2C+Jul+30%2C+2019+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/19kFKwIzXrzt7B872hKJ3nmTiuD90RzUTl7BhfF4c8a4/edit>

## Agenda

Extracted from **cross-project-council-agenda** labelled issues and pull requests from the **openjs-foundation org** prior to the meeting.

### openjs-foundation/cross-project-council

* Create page with links to slack channels and sign up link on OpenJSF.org/slack [#260](https://github.com/openjs-foundation/cross-project-council/issues/260)
* ELI5: project charters [#207](https://github.com/openjs-foundation/cross-project-council/issues/207)
* Review meeting times [#129](https://github.com/openjs-foundation/cross-project-council/issues/129)
* Update/move jsf-clabot for new fdn [#124](https://github.com/openjs-foundation/cross-project-council/issues/124)
* Post-Bootstrap work to be managed by CPC [#115](https://github.com/openjs-foundation/cross-project-council/issues/115)

## Invited

* OpenJS Foundation Cross Project Council
* OpenJS Foundation Project Maintainers
* OpenJS Foundation Board of Directors

## Notes

The agenda comes from issues labelled with `cross-project-council-agenda` across **all of the repositories in the openjs-foundation org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

link for participants: Zoom link: https://zoom.us/j/920010234
* For those who just want to watch: https://livestream.openjsf.org

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

**Comment by keywordnew** [comment]
Hey folks, has the meeting begun?

<img width="532" alt="Screen Shot 2019-07-30 at 11 36 31 AM" src="https://user-images.githubusercontent.com/10335475/62155650-4c96ea80-b2be-11e9-9a11-6a94dc8a66e8.png">

**Comment by keywordnew** [comment]
Sorry I missed the meeting. Mixed up the time.
